src/main/webapp/AlcoholPoisoning.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+
+<head>
+	<title>
+		Alcohol Poisoning
+	</title>
+	<link rel="stylesheet" href="style_emergancy.css">
+	<div class="head-style">
+		FUTURE STYLE
+	</div>
+</head>
+
+<body>
+	<!-- Breathing Content -->
+	<div class="heading">
+		<b>Alcohol Poisoning</b>
+	</div>
+	
+	<div class="sub-heading">
+		<i><b>What Is It And How Can You Help?</b></i>
+	</div>
+	
+	<div class="body-text">
+		Alcohol abuse occurs when a person drinks too much alcohol. It can cause serious medical problems that can be lethal, especially if mixed with other
+		depressants. An alcohol overdose, or poisoning, usually occurs after years of abuse and a persons tolerance becomes too high for their body to handle.
+		However, it can also happen suddenly and often occurs at parties after a person drink too much too fast. </br><b>The signs and symptoms of 
+		alcohol poisoning include:</b>
+			
+		<div class="bullet">

```

Comment:
Bulleted lists like this are supported directly with semantic HTML unordered lists: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul

It's generally preferred to use semantic HTML elements when they are available.

Reply:
I will change in future edits